tests/checks/indent.fish

```diff
@@ -96,10 +95,8 @@ end
 ' | $fish_indent
 
 #CHECK: switch aloha
-#CHECK: 
 #CHECK: {{    }}case alpha
 #CHECK: {{    }}{{    }}echo sup
-#CHECK: 
```

Comment:
I have to admit I am surprised by this change.

I'm not sure why it happens (do we synthesize an "end" token for this?), but honestly I'm okay with it.